modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,116 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity with SLA tiering. Policies control access based on the following access structuring:
+
+* Resource-level policies
+Impose a limit on all requests that match the resource to which it is applied. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+* SLA-based policies
```

Comment:
Is SLA based a policy level granularity or a configuration level? because it doesn't change where the policy is applied, just the configuration ( a.k.a limit)

Reply:
Ok! Thanks! Made a note!

Reply:
Done! Thanks!